server/authmanager.go

```diff
@@ -108,18 +106,30 @@ func (a *JWTAuthManager) stop() {
 func (a *JWTAuthManager) replaceJWT(privateKey *rsa.PrivateKey, kid string, jwtExpiration time.Duration) error {
 	log.Debugf("setting internal JWT")
 
-	token, err := NewToken(jwtExpiration)
+	rsaSigner, err := jose.NewSigner(
```

Comment:
go doesn't recommend this type of line breaking.

It should instead be
```
rsaSigner, err := jose.NewSigner(jose.SigningKey{
    Algorithm: joes.RS256,
    ...
}, ...)
```

Reply:
done